src/Sql/dbo/Stored Procedures/User_UpdateKeys.sql

```diff
@@ -15,8 +17,8 @@ BEGIN
         [Key] = @Key,
         [PrivateKey] = @PrivateKey,
         [RevisionDate] = @RevisionDate,
-        [AccountRevisionDate] = @RevisionDate,
-        [LastKeyRotationDate] = @RevisionDate
+        [AccountRevisionDate] = ISNULL(@AccountRevisionDate, @RevisionDate),
```

Comment:
AccountRevisionDate is not nullable, but the proc needs to be backwards compat with nullable default.